deploying-daml/daml-on-fabric/fabric_dependencies.md

```diff
@@ -0,0 +1,11 @@
+Now we'll need to install some Fabric dependencies (SBT), using SDKMan for convenience. Typically you also need a [few other dependencies](https://github.com/digital-asset/daml-on-fabric#prerequisites) but we already have those available in this environment.
+
```

Comment:
Add some info about when the process is done. My best guess is that it's done when you see 

`$ sdk use sbt 1.2.8

Using sbt version 1.2.8 in this shell.`

Reply:
I think for processes that exit it's not necessary to let people know when it's done, but agree for the long running daemons.